envoy/http/codec.h

```diff
@@ -492,6 +492,10 @@ struct Http1Settings {
 
   // If true, Envoy will send a fully qualified URL in the firstline of the request.
   bool send_fully_qualified_url_{false};
+
+  // If true, BalsaParser is used for HTTP/1 parsing; if false, http-parser is
+  // used.
+  bool use_balsa_parser_{false};
```

Comment:
Do we have an issue tracking migration and deprecation?   It'd be nice to link sometihng both here and in the API comments.

Reply:
Yes, #21245.  Thanks for the reminder.